testsuite/src/commonMain/kotlin/komem/litmus/LitmusTestExtensions.kt

```diff
@@ -0,0 +1,6 @@
+package komem.litmus
+
+import komem.litmus.generated.LitmusTestRegistry
+
+val LitmusTest<*>.name get() = LitmusTestRegistry.resolveName(this)
+val LitmusTest<*>.javaClassName get() = name.replace('.', '_')
```

Comment:
rename this file to `Utils.kt` ?

Reply:
Nope, apparently this clashes with another Utils.kt file from `:core`

Reply:
Can you put it in `core` then?

Reply:
No, because `core` does not contain the tests anymore, and hence this registry would be empty